camayoc/tests/qpc/api/v1/sources/test_network_sources.py

```diff
@@ -213,3 +213,76 @@ def test_negative_update_invalid(
     assert_source_update_fails(original_data, src)
     src.hosts = ['1**2@33^']
     assert_source_update_fails(original_data, src)
+
+
+def test_create_empty_str_host_valid(shared_client, cleanup):
+    """Create a host manager source and then add host data with empty string.
+
+    :id: 4BFBF5B0-81BD-48A3-8C57-BAE55590285D
```

Comment:
Nitpick, can you use lowercase for all test case ids? By doing that the id will be consistent with the other tests